.github/actions/compile-stellar/action.yaml

```diff
@@ -37,9 +37,9 @@ runs:
       run: |
         mkdir -p ./stellar-binaries
         cp "$HOME/.cargo/bin/stellar" ./stellar-binaries/stellar
-        cp "$HOME/.cargo/bin/soroban" ./stellar-binaries/soroban
+        cp "$HOME/.cargo/bin/soroban" ./stellar-binaries/soroban | true
```

Comment:
I guess we can just remove `soroban` cli completely

Reply:
my idea was to account for previous releases that still include this cli